apps/evm/src/ui/pool/AddSectionReviewModalLegacy.tsx

```diff
@@ -390,12 +397,38 @@ export const AddSectionReviewModalLegacy: FC<AddSectionReviewModalLegacyProps> =
             <>
               <DialogTrigger asChild>{children}</DialogTrigger>
               <DialogContent>
-                <DialogHeader>
-                  <DialogTitle>Add liquidity</DialogTitle>
-                  <DialogDescription>
-                    Please review your entered details.
-                  </DialogDescription>
-                </DialogHeader>
+                <div className="flex justify-between">
+                  <DialogHeader>
+                    <DialogTitle>Add liquidity</DialogTitle>
+                    <DialogDescription>
+                      Please review your entered details.
+                    </DialogDescription>
+                  </DialogHeader>
+                  <SettingsOverlay
+                    options={{
+                      slippageTolerance: {
+                        storageKey: 'addLiquidity',
```

Comment:
This is probably not this-PR-specific, but would be nice to add an enum / at least a const string to get this typed. So that it's connected to the `useSlippageTolerance` hook